CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/ProcessingResult.java

```diff
@@ -2,9 +2,24 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 class ProcessingResult {
-  List<DocumentInfo> toUpdate    = new ArrayList<>();
+  private final static String NL = System.lineSeparator();  
+
+  
+  List<SolrDocument> toUpdate    = new ArrayList<>();
   List<RecordInfo> toPostProcess = new ArrayList<>();
+  
+  public String toString() {
+    StringBuffer buf = new StringBuffer();
+    buf.append("Solr Documents:" + NL);
+    buf.append(toUpdate.stream().map(SolrDocument::toString)
+        .collect(Collectors.joining(NL)) + NL);
+    buf.append("Records:" + NL);
+    buf.append(toPostProcess.stream().map(RecordInfo::toString)
+        .collect(Collectors.joining(NL)) + NL);
```

Comment:
Style thought:

It would probably be a bit cleaner/faster to use the string builder to do the `NL` concats instead of the `+` operator.

Like instead of `buf.append("Records:" + NL);` it would be `buf.append("Records:").append(NL);`